src/components/BottomBar/index.js

```diff
@@ -49,33 +49,50 @@ export default class BottomBar extends Component {
 }
 
 const BlockInfo = withStyles(styles)(({ classes, blockNum, blockTime }) => (
-  <Grid item xs={12} md={6} className={classes.bottomBarBlockInfoWrapper}>
-    <Typography variant="body1">
-      <span className={classes.bottomBarBlockNum}><FormattedMessage id="bottomBar.blockNum" defaultMessage="Current Block Number" />{`: ${blockNum}`}</span>
-      <FormattedMessage id="bottomBar.blockTime" defaultMessage="Current Block Time" />: {blockTime ? getShortLocalDateTimeString(blockTime) : ''}
-    </Typography>
+  <Grid container item xs={12} md={6} className={classes.bottomBarBlockInfoWrapper}>
+    <Grid item xs={12} sm={5}>
+      <Typography variant="body1" className={classes.bottomBarTxt}>
+        <span className={classes.bottomBarBlockNum}><FormattedMessage id="bottomBar.blockNum" defaultMessage="Current Block Number" />{`: ${blockNum}`}</span>
+      </Typography>
+    </Grid>
+    <Grid item xs={12} sm={7}>
+      <Typography variant="body1" className={classes.bottomBarTxt}>
+        <span className={classes.bottomBarBlockTime}><FormattedMessage id="bottomBar.blockTime" defaultMessage="Current Block Time" />: {blockTime ? getShortLocalDateTimeString(blockTime) : ''}</span>
```

Comment:
The `Current Block Time` should be on the same line at sizes > XS

![image](https://user-images.githubusercontent.com/4350404/45741007-c1ba7d80-bc20-11e8-96cf-fa57c980fb9f.png)

Reply:
I developed in Chinese before, not paid attention to English. I'll pay more attention to English next time.

Reply:
Yes please. Make sure all the changes look good in all languages.